opds.py

```diff
@@ -1155,11 +1155,15 @@ def create_entry(self, work):
         identifier, work = work
 
         # Most of the time we can use the cached OPDS entry for the
-        # work.  However, that cached OPDS feed is designed around one
+        # Work. However, that cached OPDS feed is designed around one
         # specific LicensePool, and it's possible that the client is
-        # asking for a lookup centered around a different LicensePool.
+        # asking for a lookup centered around a different edition of the
+        # same book.
         default_licensepool = self.annotator.active_licensepool_for(work)
-        active_licensepool = identifier.licensed_through
+        if identifier.licensed_through:
+            active_licensepool = identifier.licensed_through[0]
+        else:
+            active_licensepool = default_licensepool
```

Comment:
The OPDS side of things is going to need some more work as well, once we actually have multiple LicensePools per Work.